docs/Install.md

```diff
@@ -0,0 +1,56 @@
+# Install
+
+Inductiva package is simple to install, just run on your terminal:
+
+```
+pip install --upgrade inductiva
+```
+
+This will provide the core functionalities of the API, which allows you to submit jobs, control machines and run simulations. To use the visualization and post-processing tools, you need to install additional optional dependencies specific to different scientific domains: `molecules_extra`, `fluids_extra` or `coastal_extra`. For example, for fluid dynamics:
+
+```
+pip install --upgrade "inductiva[fluids_extra]"
+```
+
+For any issues with the installation follow the next steps.
+
+## Installation troubleshooting
+
+### Why can't I install the optional packages?
+Depending on your shell, you may encounter issues when trying to install optional packages such as `inductiva[molecules_extra]`. This is because certain shells interpret brackets, like those in `[molecules_extra]`, in a special way. To prevent any misinterpretation or errors, enclose the package name and its extras in double quotes. To ensure a successful installation, please use the following command:
```

Comment:
This is not true anymore, is it?
We don't have these extras anymore, right?
@IvanPombo can you confirm?

Reply:
Yes, this can be removed confidently!